backend/src/routes/BankingRoutes.ts

```diff
@@ -1,10 +1,63 @@
 import { IPublicToken } from '@src/models/Banking';
 import { IReq, IRes } from './types/express/misc'
 
+require('dotenv').config();
+const { Configuration, PlaidApi, Products, PlaidEnvironments} = require('plaid');
 
 
+const APP_PORT = process.env.APP_PORT || 8000;
+const PLAID_CLIENT_ID = process.env.PLAID_CLIENT_ID;
+const PLAID_SECRET = process.env.PLAID_SECRET;
+const PLAID_ENV = process.env.PLAID_ENV || 'sandbox';
+
+// PLAID_PRODUCTS is a comma-separated list of products to use when initializing
+// Link. Note that this list must contain 'assets' in order for the app to be
+// able to create and retrieve asset reports.
+const PLAID_PRODUCTS = (process.env.PLAID_PRODUCTS || Products.Transactions).split(
+  ',',
+);
+
+
+// PLAID_COUNTRY_CODES is a comma-separated list of countries for which users
+// will be able to select institutions from.
+const PLAID_COUNTRY_CODES = (process.env.PLAID_COUNTRY_CODES || 'US').split(
+  ',',
+);
+
+// Initialize the Plaid client
+// Find your API keys in the Dashboard (https://dashboard.plaid.com/account/keys)
+
+const configuration = new Configuration({
+  basePath: PlaidEnvironments[PLAID_ENV],
+  baseOptions: {
+    headers: {
+      'PLAID-CLIENT-ID': PLAID_CLIENT_ID,
+      'PLAID-SECRET': PLAID_SECRET,
+      'Plaid-Version': '2020-09-14',
+    },
+  },
+});
+
+const client = new PlaidApi(configuration);
+
+// Retrieve the Link Token from Plaid
 async function get(_: IReq, res: IRes) {
-    res.send("I am outputing a Link Token");
+    //res.send("I am outputing a Link Token");
+
+    const configs = {
+      user: {
+        // This should correspond to a unique id for the current user.
+        client_user_id: 'user-id',
```

Comment:
Given the comment above, this should be unique per user. Let's just generate a new UUID for each request to simulate this.